### PR TITLE
new css, added css link to html

### PR DESCRIPTION
**Pull request by jacahn**
comfort: 2
completeness: 2

Had some family issues today that had me start on this assignment very late (no excuse, I know), but I did the best I could. I thought I knew css ok, but class today made me realize there is WAY more to learn. And I struggled w the basics tonight. Something to work on.

Also, for some reason, my <link rel type href>s always seem to mess up and never link properly. I need help with this.

### CODE REVIEW

**Comment by jacahn** [comment]
For some reason, some html I wrote in my last not did not show up properly.

"Also, for some reason, my s always seem to mess up and never link properly. I need help with this."

should be this:

Also, for some reason, my link rel type hrefs always seem to mess up and never link properly. I need help with this.

**Comment by jacahn** [comment]
The latter part of my previous note should state:

Also, for some reason, my LINKS REL TYPE HREF always seem to mess up and never link properly. I need help with this.

**Comment by jacahn** [comment]
I'll get the hang of this someday. lol

File: index.html [hunk 1]
```diff
@@ -2,14 +2,15 @@
 <head>
   <title>Wendy Bite | About Me</title>
   <meta charset="utf8">
+  <link rel="stylesheet" type="text/css" href="theme.css">
 </head>
```
Comment (andrewsunglaekim) [inline index.html]: The reason its messing up is that the href attribute(ie. `href="theme.css"`) needs to be `href="styles.css"` the same exact name as your file.

**Comment by andrewsunglaekim** [comment]
Definitely give this exercise another shot when you get a chance.

**Pull request closed without merging**